src/components/Agreements/ViewAgreement/Sections/LicenseBusinessTerms.js

```diff
@@ -1,25 +1,53 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { get } from 'lodash';
 import { FormattedMessage } from 'react-intl';
 import { Accordion } from '@folio/stripes/components';
+import { LicenseTermsList } from '@folio/stripes-erm-components';
 
 export default class LicenseBusinessTerms extends React.Component {
   static propTypes = {
+    linkedLicenses: PropTypes.shape({
```

Comment:
`linkedLicenses` is inside `agreement`, as that's where we pull it from on L36. This should be something like:
```
agreement: PropTypes.shape({
  linkedLicenses: PropTypes.arrayOf(PropTypes.shape({
    remoteId_object: ...
    ... 
  }))
}
```

Let's get this fixed since as-is, it's suggesting something incorrect to us and other developers.

(While we're at it, let's alpha-sort the `linkedLicenses` properties. Ie, `remoteId_object` before `status`)